cabal-testsuite/PackageTests/Check/ConfiguredPackage/COptions/CCOptionsExtraLibDirs/cabal.out

```diff
@@ -1,4 +1,4 @@
 # cabal check
-Warning: The following warnings are likely to affect your build negatively:
-Warning: Instead of 'cc-options: -Llibdir' use 'extra-lib-dirs: libdir'
-Warning: Hackage would reject this package.
+The following warnings are likely to affect your build negatively:
```

Comment:
It's strange that it says "The following warnings..." and then gives a list of "Error"s. I'm not sure what to do about it. Maybe instead of "Error" it should print the actual problem first and then add something like `-- suspected Hackage error!`...

Reply:
Or just don't call them warnings in "The following...". Call them issues or some such...

Reply:
Done!